w1r3/java/src/main/java/W1R3.java

```diff
@@ -139,13 +126,45 @@ public Integer call() throws Exception {
             Executors.newScheduledThreadPool(
                 1, new ThreadFactoryBuilder().setNameFormat("worker-%d").build()));
 
-    try (var otelSdk = setupOpenTelemetrySdk(instance)) {
+    var baseTracingAttributes =
+        Attributes.builder()
+            .put("ssb.language", "java")
+            .put("ssb.deployment", deployment)
+            .put("ssb.instance", instance)
+            .put("ssb.region", region)
+            .put("ssb.version", "unknown")
+            .put("ssb.version.sdk", SDK_VERSION)
+            .put("ssb.version.grpc", GRPC_VERSION)
+            .put("ssb.version.protobuf", PROTOBUF_VERSION)
+            .put("ssb.version.http-client", HTTP_CLIENT_VERSION)
+            .build();
+
+    var baseMeterAttributes =
```

Comment:
Feels like we could compute `baseMeterAttributes` from `baseTracingAttributes` or something.